Lib/ipaddress.py

```diff
@@ -1834,8 +1854,23 @@ def __init__(self, address):
         addr_str = str(address)
         if '/' in addr_str:
             raise AddressValueError("Unexpected '/' in %r" % address)
+        addr_str, self._scope_id = self._split_scope_id(addr_str)
+
         self._ip = self._ip_int_from_string(addr_str)
 
+    def __str__(self):
+        ip_str = super().__str__()
+        return ip_str + '%' + self._scope_id if self._scope_id else ip_str
+
+    @property
+    def scope_id(self):
+        """Identifier of a particular zone of the address's scope.
+
+        Returns:
+            A string, identifying the zone of the address if specified, else None.
```

Comment:
Small nit: delete the comma in "A string identifying ...". Otherwise it reads as though it's always a string.

Reply:
[Fixed](https://github.com/python/cpython/pull/13772/commits/ac19fadfbcf73e314816c15c05f110cceede5b73)